boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20-common.dtsi

```diff
@@ -21,3 +21,19 @@
 	startup-time-us = <600000>;
 	mode = "crystal";
 };
+
+&fll16m {
+	status = "okay";
+	open-loop-accuracy-ppm = <20000>;
+	closed-loop-base-accuracy-ppm = <5000>;
+	hfxo-clock = <&hfxo>;
+	lfxo-clock = <&lfxo>;
```

Comment:
why is this modified at board level?

Reply:
also, this should be:

```
clocks = <&hfxo>, <&lfxo>;
clock-names = "hfxo", "lfxo";
```

Reply:
Changed to use clocks property, and moved to the common nrf54h20.dtsi